tests/unittest/PinAssignment/PortsLoader_test.cpp

```diff
@@ -68,3 +70,24 @@ TEST(PortsLoader, LoadAllData) {
     EXPECT_EQ(port3.range, "Msb: 2, lsb: 0") << port3.range.toStdString();
   }
 }
+
+TEST(PortsLoader, LoadBus) {
+  PortsModel model;
+  PortsLoader loader{&model};
+  auto [res, error] = loader.load(":/PinAssignment/ports_test.json");
```

Comment:
Can you link me to this json ? I didn't find it under /src/PinAssignment.

Reply:
it is not under src, https://github.com/os-fpga/FOEDAG/blob/main/tests/unittest/PinAssignment/ports_test.json